spec/classes/foreman_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe 'foreman' do
   on_supported_os.each do |os, facts|
     context "on #{os}" do
-      let(:facts) { facts }
+      let(:facts) { override_facts(facts, processors: { count: 3 }, memory: { system: { total_bytes:  10737418240}}) }
```

Comment:
In Ruby it's valid to write:
```suggestion
      let(:facts) { override_facts(facts, processors: { count: 3 }, memory: { system: { total_bytes: 10_737_418_240}}) }
```

I think that makes it easier to spot it's 10 GB.